index.js

```diff
@@ -160,6 +160,12 @@ Hbs.prototype.createRenderer = function() {
     var tplPath = hbs.getTemplatePath(tpl),
       template, rawTemplate, layoutTemplate;
 
+    if(tplPath === undefined) {
+      this.status = 500;
```

Comment:
I meant to mention, there won't be a release with this change. There will be a new minor version once #42 closes.